### ISSUE TITLE
Code gen package name contains invalid keyword

### ISSUE THREAD

**Issue by macroadster** [softlayer/softlayer-java#38]
When compiling with maven 3.3.9 and Java 1.8.0_111, the compilation fails with:

```
[INFO] --- maven-compiler-plugin:3.0:compile (default-compile) @ softlayer-api-client ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 1084 source files to /Users/eric/s/softlayer-java/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /Users/eric/s/softlayer-java/src/main/java/com/softlayer/api/service/container/authentication/response/2FactorAuthenticationNeeded.java:[13,13] <identifier> expected
[ERROR] /Users/eric/s/softlayer-java/src/main/java/com/softlayer/api/service/container/authentication/response/2FactorAuthenticationNeeded.java:[13,41] <identifier> expected
[ERROR] /Users/eric/s/softlayer-java/src/main/java/com/softlayer/api/service/container/authentication/response/2FactorAuthenticationNeeded.java:[13,56] <identifier> expected
[ERROR] /Users/eric/s/softlayer-java/src/main/java/com/softlayer/api/service/container/authentication/request/native/external/Totp.java:[1,68] <identifier> expected
```
The package name generated from code gen process is using:
```
package com.softlayer.api.service.container.authentication.request.**native**.external;
```

Native is a keyword in Java.  Java package name convention prevents usage of Java keyword.  This is causing compilation to fail.

**Comment by camporter** [comment]
Pushed a fix in PR #39 for the package naming issue mentioned here and an issue with a class name that started with a number.

**Comment by nchiluku** [comment]
It doesn't still fix the issue. Here is what I see:
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 9.110 s
[INFO] Finished at: 2016-11-16T11:46:41-08:00
[INFO] Final Memory: 17M/355M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.0:compile (default-compile) on project softlayer-api-client: Compilation failure: Compilation failure:
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/container/authentication/request/native/external/Verisign.java:[1,68] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/container/authentication/request/native/external/Verisign.java:[1,74] class, interface, or enum expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/container/authentication/request/native/external/Verisign.java:[1,75] class, interface, or enum expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/container/authentication/request/native/external/Verisign.java:[5,67] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/container/authentication/request/native/external/Verisign.java:[5,73] class, interface, or enum expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/container/authentication/request/native/external/Verisign.java:[5,74] class, interface, or enum expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/container/authentication/request/native/external/Verisign.java:[84,97] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/container/authentication/request/native/external/Verisign.java:[84,103] illegal start of type
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/container/authentication/request/native/external/Verisign.java:[84,112] ';' expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/container/authentication/request/native/external/Verisign.java:[84,117] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[931,64] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[931,70] ')' expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[931,71] class, interface, or enum expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[931,85] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[932,64] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[932,70] ';' expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[932,85] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[933,8] ';' expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[933,64] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[933,70] ';' expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[933,89] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[934,8] ';' expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[934,71] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[935,8] ';' expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[935,71] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[936,8] ';' expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[936,87] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[937,8] ';' expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[937,92] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[938,8] ';' expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[938,96] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[939,8] ';' expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[939,78] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[940,8] ';' expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[940,73] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[941,8] ';' expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[941,64] <identifier> expected
[ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[949,8] ';' expected
219 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[949,65] <identifier> expected
220 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[950,8] ';' expected
221 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[950,63] <identifier> expected
222 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[951,8] ';' expected
223 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[951,62] <identifier> expected
224 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[952,8] ';' expected
225 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[952,55] <identifier> expected
226 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[953,8] ';' expected
227 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[953,71] <identifier> expected
228 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[954,8] ';' expected
229 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[954,79] <identifier> expected
230 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[955,8] ';' expected
231 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[955,77] <identifier> expected
232 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[956,8] ';' expected
233 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[956,63] <identifier> expected
234 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[957,8] ';' expected
235 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[957,82] <identifier> expected
236 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[958,8] ';' expected
237 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[958,75] <identifier> expected
238 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[959,8] ';' expected
239 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[959,79] <identifier> expected
240 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[960,8] ';' expected
241 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[960,63] <identifier> expected
242 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[961,8] ';' expected
243 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[961,70] <identifier> expected
244 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[962,8] ';' expected
245 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[962,82] <identifier> expected
246 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[963,8] ';' expected
247 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[963,68] <identifier> expected
248 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[964,8] ';' expected
249 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[964,83] <identifier> expected
250 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[965,8] ';' expected
251 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[965,75] <identifier> expected
252 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[966,8] ';' expected
253 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[966,75] <identifier> expected
254 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[967,8] ';' expected
255 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[967,78] <identifier> expected
256 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[968,8] ';' expected
257 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[968,96] <identifier> expected
258 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[969,8] ';' expected
259 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[969,104] <identifier> expected
260 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[970,8] ';' expected
261 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[970,84] <identifier> expected
262 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[971,8] ';' expected
263 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[971,82] <identifier> expected
264 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[972,8] ';' expected
265 [ERROR] /softlayer-java/src/main/java/com/softlayer/api/service/package-info.java:[972,77] <identifier> expected
266 [ERROR] -> [Help 1]
267 [ERROR]
268 [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
269 [ERROR] Re-run Maven using the -X switch to enable full debug logging.

**Comment by nchiluku** [comment]
By using mvn eclipse:eclipse, it compiles but it has lot of errors in eclipse too:
![eclipseerrors](https://cloud.githubusercontent.com/assets/23508703/20363312/9edfb40e-abf3-11e6-9375-e58575a6c1df.png)

**Comment by camporter** [comment]
Are you on the branch listed in the pull request above? Have you tried running `mvn clean` first and then tried `mvn compile`?

**Comment by nchiluku** [comment]
1)I pulled and compiled in July, using mvn eclipse:eclipse. I had no problems importing it, in eclipse. We need it because of the defect listed in http://stackoverflow.com/questions/38381633/softlayer-os-reload-api
2)#macroadster was trying to have a daily build, so he tried to build yesterday.  Since he was not successful, I tried it too. 
3) Today, I tried it on my local machine using git clone -b master https://github.com/softlayer/softlayer-java.git as mentioned http://stackoverflow.com/questions/38381633/softlayer-os-reload-api. It failed with the above errors.

**Comment by macroadster** [comment]
@camporter : One of the java file is generated with filename "2FactorAuthentication.java".  This is not a valid java class name neither.  Could you also fix this issue?

**Comment by camporter** [comment]
The fix from the pull request is on the master branch now, so please try again.

**Comment by nchiluku** [comment]
I pulled it and it worked for me. macroadster will test it when he finds time. 
Thanks a lot! Are you going to have a official release any time soon with all these fixes?

**Comment by camporter** [comment]
I've prepped changes for a new minor release and set up credentials for maven central. @cretz do you see any issue with either transferring ownership of the groupId or requesting addition of 'cmporter' to the groupId?

**Comment by cretz** [comment]
@camporter - Nope. Be advised, I no longer have access to any of that Sonatype stuff so I can not do any of it for you.

**Comment by camporter** [comment]
@cretz thanks for your help. I'll try and get access and get a new release out.

**Comment by camporter** [comment]
Released 0.2.3 which should address this issue.

**Comment by nchiluku** [comment]
Thank you so much!
